projects/gameboard-ui/src/app/api/game.service.ts

```diff
@@ -132,17 +130,17 @@ export class GameService {
     game.cardUrl = game.logo
       ? `${this.config.imagehost}/${game.logo}`
       : `${this.config.basehref}assets/card.png`
-    ;
+      ;
```

Comment:
Is this linter as well? Seems like a dangling semi-colon should indent to the instruction it is completing.